docs/UserGuide.md

```diff
@@ -569,57 +571,37 @@ Acceptable values for `e/EMAIL`:
-2. This is followed by a '@' and then a domain name. The domain name is made up of domain labels separated by periods.
-   The domain name must:
-    - end with a domain label at least 2 characters long
-    - have each domain label start and end with alphanumeric characters
-    - have each domain label consist of alphanumeric characters, separated only by hyphens, if any.".
-
-Acceptable values for `a/ADDRESS`:
-- Word with or without spaces.
-- Error message for invalid address: "Addresses can take any values, and it should not be blank".
-
-Acceptable values for `r/RSVP_STATUS`:
+
+**Acceptable values for `a/ADDRESS`:**
+- Word with or without spaces and should not be blank.
+
+**Acceptable values for `r/RSVP_STATUS`:**
 - `yes`
 - `no`
 - `unknown`
 - Inputs with no values when adding a guest (e.g. `r/`) signify that RSVP status should be stored as `unknown`.
 - Inputs with no values when filtering guests (e.g. `r/`) signify that you wish to filter for guests with `unknown` RSVP status.
-- Error message for invalid RSVP status: “RsvpStatus Status should only have one of three values. yes, no, or unknown.”.
 
-Acceptable values for `d/DIETARY_REQUIREMENT`:
+**Acceptable values for `d/DIETARY_REQUIREMENT`:**
 - Alphanumeric word with or without spaces and should not be blank.
-- Error message for invalid table numbers: "Dietary requirements should only contain alphanumeric characters and spaces, and it should not be blank"
 
-Acceptable values for `tn/TABLE_NUMBER`:
+**Acceptable values for `tn/TABLE_NUMBER`:**
 - Numbers between 1 and 2147483647 inclusive with no spaces or special characters
 - Preceding zeros will be trimmed.
-- Error message for invalid table numbers: "Table numbers should only contain numbers"
 
-Acceptable values for `t/TAG`:
+**Acceptable values for `t/TAG`:**
 - Alphanumeric word without spaces.
-- Error message for invalid tags: "Tags names should be alphanumeric"
 
 --------------------------------------------------------------------------------------------------------------------
 
-### 7.2. Appendix B: Miscellaneous error messages
+### 7.2. Appendix B: Miscellaneous errors
 
-User input is completely invalid (e.g. `abc` or `vsdf`):
-- Display error message "Unknown command.".
-  <br />
-User input begins with `vendor` or `guest`, but does not include a valid command word (e.g. `vendor abc` or `guest adddd`):
-- Display error message:
-  <br />
-"Invalid command format!
-<br />
-help: Shows program usage instructions.
-<br />
-Example: help".
+**User input is completely invalid (e.g. `abc` or `vsdf`):**
+- A valid command instruction is required.
 
 <br />
 
+**User input begins with `vendor` or `guest`, but does not include a valid command word (e.g. `vendor abc` or `guest adddd`):**
+- A valid command instruction after `vendor` or `guest` is required.
+
```

Comment:
Not sure if this appendix is necessary anymore?

Reply:
Have deleted the appendix and areas that have referred to it, awaiting others response